packaging/installer/kickstart.sh

```diff
@@ -27,7 +27,7 @@ KICKSTART_SOURCE="$(
 )"
 PACKAGES_SCRIPT="https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/install-required-packages.sh"
 PATH="${PATH}:/usr/local/bin:/usr/local/sbin"
-PUBLIC_CLOUD_URL="https://app.netdata.cloud"
+PUBLIC_CLOUD_URL="https://api.netdata.cloud"
```

Comment:
I think this is not correct, as `PUBLIC_CLOUD_URL` is also used in this script to point to the web app in a `printf`.